src/locales/ar.po

```diff
@@ -3229,9 +3250,7 @@ msgid "Win payout if %3 is strictly higher than %6 at %5 after %4."
 msgstr "أكسب عائد إذا ما كانت ال %3 أعلى بدقة من %6 عند %5 بعد %4."
 
 msgid "Win payout if %3 after %plural(%5,%d tick,%d ticks) is strictly higher than %6."
-msgstr ""
-"أكسب عائد إذا ما كانت ال %3 بعد %plural(%5,%d علامة,%d علامات) بدقة أعلى من "
-"%6."
+msgstr "أكسب عائد إذا ما كانت ال %3 بعد %plural(%5,%d علامة,%d علامات) بدقة أعلى من %6."
```

Comment:
I'm not sure about this one, but the rendering in github at least looks quite confused, might be some stray RTL/LTR chars in there?